### PR TITLE
feat: Define the interface for the orderbook matches

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: crates/orderbook-commons/Cargo.toml [hunk 1]
```diff
@@ -11,3 +11,5 @@ secp256k1 = { version = "0.24.3", features = ["serde"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 sha2 = { version = "0.10", default-features = false }
+time = "0.3.20"
+uuid = { version = "1.3.0", features = ["v4", "fast-rng", "macro-diagnostics", "serde"] }
```
Comment (bonomat) [inline crates/orderbook-commons/Cargo.toml:15]: do we need all features?

File: crates/orderbook-commons/src/lib.rs [hunk 2]
```diff
@@ -21,6 +24,68 @@ pub fn create_sign_message() -> Message {
...
+    /// The oracle_pk is used to define what oracle is to be used in the contract.
+    /// This `oracle_pk` must correspond to one `oracle_pk` configured in the dlc-manager.
+    /// It is possible to configure multiple oracles in the dlc-manager; this
+    /// `oracle_pk` has to match one of them. This allows us to configure the dlc-managers
+    /// using two oracles, where one oracles can be used as backup if the other oracle is not
+    /// available. Eventually this can be changed to be a list of oracle PKs and a threshold of
+    /// how many oracle have to agree on the attestation.
+    pub oracle_pk: XOnlyPublicKey,
+
+    /// The matches for the
```
Comment (bonomat) [inline crates/orderbook-commons/src/lib.rs:85]: Comment incomplete :)

**Review by bonomat — APPROVED** [review]

File: crates/orderbook-commons/Cargo.toml (hunk 1 above)
Comment (da-kami) [reply crates/orderbook-commons/Cargo.toml:15]: Sorry, copy paster, I'll remove the ones we don't need

**Pull request merged**